trin-history/src/validation.rs

```diff
@@ -278,30 +202,9 @@ mod tests {
             .unwrap();
     }
 
-    #[tokio::test]
-    async fn validate_block_body() {
-        let server = setup_mock_infura_server();
-
-        let ssz_block_body: Vec<u8> =
-            std::fs::read("../test_assets/mainnet/block_body_14764013.bin").unwrap();
-        let block_body_bytelist: VariableList<_, typenum::U16384> =
-            VariableList::from(ssz_block_body);
-
-        let header_oracle = default_header_oracle(server.url("/14764013"));
-        let chain_history_validator = ChainHistoryValidator { header_oracle };
-        let content_key = block_14764013_body_key();
-
-        chain_history_validator
-            .validate_content(&content_key, &block_body_bytelist)
-            .await
-            .unwrap();
-    }
-
     #[tokio::test]
     #[should_panic]
     async fn invalidate_block_body() {
```

Comment:
I do think if we're removing the `validate_*` tests then we also ought to remove the `invalidate_*` tests since the `invalidate_*` tests aren't really proving anything useful anymore, now that we've removed the trusted provider. 

However, I think it's important that we don't simply remove the `validate_*` tests, but update them to test cases that we can actually validate. The only useful validation test that we can do at this point is to validate pre-merge headers (with proofs) against the accumulator. 

Imo, the easiest way to test this case would be to add a `validation.rs` module to `ethportal-peertest`. Add a scenario for validating a pre-merge header-with-proof, bb, & receipts. That way, we can seed a node with the validation data needed to perform the validation with local data vs. needing to reach out into the network. Actually, it would also enable us to have a scenario where the node needs to fetch validation data from a peer, which would be a nice test. 

Anyways, this is starting to feel like it belongs in a separate tracking issue & pr. For this specific pr, I'd just remove all of the invalidate/validate tests for block bodies & receipts.